composer.json

```diff
@@ -7,7 +7,7 @@
     "require": {
         "composer/installers": "~1.2",
         "tecnickcom/tcpdf": "^6.2",
-        "stripe/stripe-php": "^6.41"
+        "stripe/stripe-php": "^7.44.0"
```

Comment:
@ravinderk I updated the Stripe PHP SDK because it was throwing errors. Do you foresee any complications with updated the Stripe PHP SDK?

Reply:
@kjohnson Please revert is as we discussed on slack: https://givewp.slack.com/archives/C0FAGC83C/p1595604020123300